leapp/libraries/stdlib/call.py

```diff
@@ -37,17 +37,18 @@ def _multiplex(ep, read_fds, callback_raw, callback_linebuffered,
     while not ep.closed and len(hupped) != num_expected:
         events = ep.poll(timeout)
         for fd, event in events:
+            translated_fd = read_fds.index(fd) + 1
```

Comment:
We'll need to think a little bit more about this, technically this works because the index of the element in the array corresponds to the actual numerical value of STDOUT/STDERR, however the value of `translated_fd` does not represent the real file descriptor, because both `stdout` and `stderr` are actually pipes (https://github.com/oamg/leapp/blob/5d1ee15732f5be216eab60560f9682a5719ec4ca/leapp/libraries/stdlib/call.py#L108).

I'd rename `translated_fd` to `fd_type`, to better reflect what the value represents, and change the semantics of `callback_{raw,linebuffered}` to treat the first argument as a tuple -- (fd, fd_type), this way we can still distinguish whether we're dealing with `stdout` or `stderr` while retaining the access to the real file descriptor.